nym-vpn-core/nym-vpn-lib/uniffi/nym_vpn_lib.swift

```diff
@@ -1313,6 +1547,154 @@ public func FfiConverterTypeWgConfig_lower(_ value: WgConfig) -> RustBuffer {
     return FfiConverterTypeWgConfig.lower(value)
 }
 
+// Note that we don't yet support `indirect` for enums.
+// See https://github.com/mozilla/uniffi-rs/issues/396 for further discussion.
```

Comment:
Does it print this for every enum?

Reply:
I guess so. The nym_vpn_lib files are auto-generated.